### DISCUSSION TITLE
Copy current filter as query

### DISCUSSION THREAD

**Discussion by pablomedok** [Sequel-Ace/Sequel-Ace#1391] [Ideas]
Please make it possible to copy current SQL query to use it in editor. I usually start Sequel-Ace in table mode to filter the data I need to manipulate. A create a simple filter with 2-3 fields. But later I frequently understand that I need to use a subquery or to perform an update or delete. 
It would be great to have a 'Copy Query' button that would copy current filter as a query to be used in Editor.

![query](https://user-images.githubusercontent.com/13901628/153882114-d449cb64-b43f-438c-8ba8-c0eb88df7245.png)

Pressing Copy Query would copy the whole SELECT query to clipboard and in my example would produce:
`SELECT * FROM `b_crm_requisite` WHERE (`ENTITY_TYPE_ID` = '4') AND (`DATE_MODIFY` > '2022-01-01') AND (`NAME` IS NOT NULL) LIMIT 0,1000;`

Maybe without LIMIT 0,1000

I know that I can use Console to do that, but that's at least 5 extra clicks and 1 edit to delete the LIMIT. 

P.S.: Please, also replace '—' with a real button with caption 'Clear'. — doesn't look like a button.

**Comment by Jason-Morcos** [comment]
Related to: https://github.com/Sequel-Ace/Sequel-Ace/issues/290
PR is welcome to add a Copy Query button! I'd happily review it
